inter-operator/architecture/basic.md

```diff
@@ -625,3 +626,15 @@ After the interoperator is ready and setup across multiple clusters as described
 3. Interoperator does not take care of the Kubernetes and OS updates to the onboarded clusters.
 4. Service owners will have to monitor the clusters and their resource situations and add additional sister clusters if required.
 
+# Mass Update of Custom Resources for Interoperator Custom Resource changes
+
+## Context
+Interoperator has custom resources like SFPlans and SFServices which one has to provide and deploy before working with interoperator. This was already described in [here](#service-fabrik-inter-operator-custom-resources). Based on the templates defined in `SFPlan`, service specific custom resources are rendered and with that, service instance creation takes place. So, in a way, `SFPlan` and also `SFService` CRs are used as references when the service specific CRs created. However, when these reference CRs like `SFPlans` and `SFServices` change, the changes are not automatically reflected on the service specific CRs. Because of that, when a service owner changes `SFPlans` and some of its attributes and templates, older service CRs are not automatically changed.
+              The situation is similar if a service broker updates its catalog and any of it's metadata, which is used by service to configure a specific service instance. Should the broker trigger an update of all the service instances immediately or should it wait for an user initiated update operation ?
+              
+## Solution
+There are no generic guideline from the OSB spec as well and the solution to this would entirely depend on the service broker implementation. The problem with having immediate trigger of an update for all affected service instances would be that updates can cause downtime depending on how services are handling it.
```

Comment:
```suggestion
There are no generic guidelines from the OSB spec as well and the solution to this would entirely depend on the service broker implementation. The problem with having immediate trigger of an update for all affected service instances would be that updates can cause downtime depending on how services are handling it.
```